### PR TITLE
Update of the metadata file with the paper reference

### PR DESCRIPTION
**Pull request by bkhelifi**
**Description**
This pull request add the paper reference into the metadata files of the project. Should we backport this PR to the bug releases?

For the next release, one should take care to add the new added fields in the metadata files.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -19,6 +19,10 @@ license: BSD-3-Clause
 contact:
 - email: GAMMAPY-COORDINATION-L@IN2P3.FR
   name: "Coordination committee of the Gammapy project"
+identifiers:
+- description: "Zenodo repository of the release 1.1"
```
Comment (AtreyeeS) [inline CITATION.cff:23]: Should be 1.2 no?

File: CITATION [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
-  (Deil et al., 2017).
-
-where (Deil et al., 2017) is a citation to this paper:
-
-  https://ui.adsabs.harvard.edu/abs/2017ICRC...35..766D
-
-We encourage you to also include citations to the paper in the main text
-wherever appropriate.
+the following citation scheme:
+- refer to the Gammapy paper as general citation: A&A, 678, A157 (2023) - the bibtex is given bellow
```
Comment (AtreyeeS) [inline CITATION:4]: maybe its better to explicitly mention that oner should cite paper AND zenodo version, not either or

**Review by AtreyeeS — APPROVED** [review]
Thanks @bkhelifi !

I am wondering if we can also include the zenodo doi bibtex entry as well?

File: CITATION.cff (hunk 1 above)
Comment (bkhelifi) [reply CITATION.cff:23]: When it will be tagged, yes, of course... Should I do it right now, @registerrier ?

**Comment by bkhelifi** [comment]
> Thanks @bkhelifi !
> 
> I am wondering if we can also include the zenodo doi bibtex entry as well?

Of which release? the latest one, I guess... which makes no sense once a new release is made. You see my point?

**Comment by AtreyeeS** [comment]
> Of which release? the latest one, I guess... which makes no sense once a new release is made. You see my point?

The DOI [10.5281/zenodo.4701488](https://zenodo.org/doi/10.5281/zenodo.4701488) always resolves to the latest one one no?

File: CITATION (hunk 2 above)
Comment (registerrier) [inline CITATION:4]:
```suggestion
- refer to the Gammapy paper as general citation: A&A, 678, A157 (2023) - the bibtex is given below
```

File: CITATION [hunk 3]
```diff
@@ -1,34 +1,30 @@
...
+- AND refer to the used version of Gammapy cited in Zenodo with the DOI 10.5281/zenodo.4701488
```
Comment (registerrier) [inline CITATION:5]:
```suggestion
- AND cite the version of Gammapy used with the DOI 10.5281/zenodo.4701488
```

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi . Minor suggestions inline.

**Review by adonath — COMMENTED** [review]
Thanks @bkhelifi! I have no additional comments, please address the ones by @AtreyeeS  and @registerrier.

**Comment by registerrier** [comment]
Thanks @bkhelifi . Merging now.

**Pull request merged**